src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/common/category/CategoryPill.java

```diff
@@ -0,0 +1,54 @@
+package com.wikia.webdriver.elements.mercury.components.discussions.common.category;
+
+import lombok.AllArgsConstructor;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'lombok.AllArgsConstructor'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)